cmd/krel/cmd/release_notes.go

```diff
@@ -113,12 +156,122 @@ func runReleaseNotes() (err error) {
 	logrus.Infof("Using start tag %v", start)
 	logrus.Infof("Using end tag %v", tag)
 
-	_, err = releaseNotesFrom(start)
+	if releaseNotesOpts.createDraftPR {
+		err = validateDraftPROptions()
+		if err != nil {
+			return errors.Wrap(err, "validating PR command line options")
+		}
+	}
+
+	result, err := releaseNotesFrom(start)
 	if err != nil {
 		return errors.Wrapf(err, "generating release notes")
 	}
 
-	//TODO: implement PR creation for k-sigs/release-notes and k/sig-release
+	// Create RN draft PR
+	if releaseNotesOpts.createDraftPR {
+		err = createDraftPR(tag, result)
+		if err != nil {
+			return errors.Wrap(err, "Failed to create release notes draft PR")
```

Comment:
Nittest of a nit:
```suggestion
			return errors.Wrap(err, "failed to create release notes draft PR")
```
Let's start errors lower case to look homogeneous when showing them to the user.

Reply:
I was trying to understand whe or how errors where upper/lower case. Now I now. Done